kansha/app/view.py

```diff
@@ -130,6 +130,8 @@ def render(self, h, comp, *args):
     h.head.javascript_url('js/kansha.js')
     h.head.javascript_url('js/autocomplete.js')
 
+    h.head.javascript_url('//cdn.ckeditor.com/4.5.3/standard/ckeditor.js')
```

Comment:
Well, I missed that when I accepted the PR. I should not be in a hurry to merge and I should let the peer review take place longer, now we have more contributors.
@bcroq is right, we should embed the js dependencies in the package, as Kansha should remain usable in a closed intranet. 
@droodle, please, could you fix that? And exclude it from code analysis in .codeclimate.yml. Thank you.